web-entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Now that the environment variables are populated by the task definition, this is much more streamlined and generalized.